src/Take.Blip.Builder/Actions/ProcessHttp/ProcessHttpAction.cs

```diff
@@ -129,6 +134,24 @@ private void AddBotIdentityToHeaders(HttpRequestMessage httpRequestMessage, ICon
             }
```

Comment:
if "CheckInternalUris(absoluteUri)" is true, the BLIP_BOT_HEADER will be duplicated

Reply:
is good if you add a unit test for it...

Reply:
Done